automated/android/tradefed/result_parser.py

```diff
@@ -57,6 +57,11 @@ def parse(self, xml_file):
             ):
                 etree_content = etree_content[:mstart] + etree_content[mend:]
                 endpos = len(etree_content)
+            # next time search again from the same position as this time
+            # as the detected pattern was removed here
+            pos = mstart
+        else:
```

Comment:
is this properly indented? It looks like 'else' has the same indentation as 'while'. Seems weird to me as pos has no meaning outside of while loop

Reply:
sorry, it's not correct. updated already.